Firestore/Example/Tests/Util/FSTIntegrationTestCase.mm

```diff
@@ -14,10 +14,9 @@
  * limitations under the License.
  */
 
-@import Firestore;
-
 #import "FSTIntegrationTestCase.h"
 
+#import <Firestore/Firestore-umbrella.h>
```

Comment:
Likely corresponding copybara change needed.

Reply:
I'll tackle this while trying to get the import working.